cmd/git-sync/main.go

```diff
@@ -181,7 +184,9 @@ func main() {
 	initialSync := true
 	failCount := 0
 	for {
-		if err := syncRepo(*flRepo, *flBranch, *flRev, *flDepth, *flRoot, *flDest); err != nil {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*time.Duration(*flSyncTimeout))
+		err := syncRepo(ctx, *flRepo, *flBranch, *flRev, *flDepth, *flRoot, *flDest)
```

Comment:
Why did you change the style from `if err:= ...; err != nil` ?  I don't see a need for err beyond this block?

Reply:
Accidental, reverted.